app/views/users/post.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Details of post</h1>
```

Comment:
- [ ] After creating the PostsController, we need to create the corresponding view files for each of the PostsController actions, something like this; 
```ruby
     app/views/posts/index.html.erb
     app/views/posts/show.html.erb
```

so please delete these files below because we do not need them.
```ruby
    app/views/users/post.html.erb
    app/views/users/posts.html.erb
```